client/post-editor/media-modal/detail/detail-preview-image.jsx

```diff
@@ -29,7 +29,7 @@ export default class EditorMediaModalDetailPreviewImage extends Component {
 		super( props );
 
 		this.onImagePreloaderLoad = this.onImagePreloaderLoad.bind( this );
-		this.state = { loading: false };
+		this.state = { loading: false, spin: false };
```

Comment:
While Spinner isn't working properly in #39616, it's working the same as currently in production. Adding `setSpinner` works, but it's pretty specific to media modal. In general case, it seems like the same thing can be accomplished by using `onLoad` which is triggered right after `setImageObjectUrl`. In this case the following change would be enough:

```suggestion
		this.state = { loading: true };
```